app/models/facility.rb

```diff
@@ -179,9 +179,16 @@ def teleconsultation_enabled?
   end
 
   def teleconsultation_phone_number_with_isd
-    return if teleconsultation_isd_code.blank? || teleconsultation_phone_number.blank?
+    teleconsultation_phone_number = teleconsultation_phone_numbers.first
+    return if teleconsultation_phone_number.blank?
+
+    Phonelib.parse(teleconsultation_phone_number.isd_code + teleconsultation_phone_number.phone_number).full_e164
+  end
 
-    Phonelib.parse(teleconsultation_isd_code + teleconsultation_phone_number).full_e164
+  def teleconsultation_phone_numbers_with_isd
```

Comment:
There are several new behaviors on the Facility model, could we add test coverage for them?